### PR TITLE
Update nested-cluster-quickstart.md

### PR DESCRIPTION
**Pull request by antongisli**
Requires:
- https://github.com/spectrocloud/optic/pull/2979 (See: https://spectrocloud.atlassian.net/browse/PPD-194)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/883#" title="Author self-approved">antongisli</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @antongisli. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Review by Siboney1 — APPROVED** [review]
/approved

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/883#" title="Author self-approved">antongisli</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1107973081" title="Approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (resmaa) [inline content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:22]: @Siboney1 , here enable is for the host cluster not for the nested cluster

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (resmaa) [inline content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:20]: With the Palette Nested Cluster option, you can deploy additional Kubernetes clusters, within an existing cluster, to provide teams with nested cluster environments.

There are certain advantages to use a nested cluster environment. For example, you are free to operate with admin level privileges, while simultaneously ensuring strong isolation, reducing operational overhead, and improving resource utilization.

the above paragraph is information redundancy from the landing page overview. We can consider removing this @Siboney1

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (resmaa) [inline content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:28]: "During the cluster creation, you will be able to configure it as a Host Cluster."

here lets specify "During the nested cluster creation, you will be able to configure it as a Host Cluster.",  

-else it can confuse with host cluster creation @Siboney1

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (resmaa) [inline content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:37]: Refer to the [Cluster Endpoint](https://docs.spectrocloud.com/clusters/nested-clusters/cluster-quickstart#loadbalancer) step below for more information on how to configure this option. The same settings listed below are available, when you are enabling a Nested Cluster and when deploying a Host Cluster deployment wizard.


here ' this option ' refers to?
the whole paragraph is slightly confusing. @Siboney1

**Comment by resmaa** [comment]
image
image
do we need these screenshots?  can we go for video instead.? @Siboney1 @truegilb

**Review by resmaa — CHANGES_REQUESTED** [review]

**Comment by resmaa** [comment]
image
in landing page, **LoadBalancer** can be **Load Balancer**

**Comment by resmaa** [comment]
Should we add "Beta" to the page title, as UI has it as of today

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (Siboney1) [reply content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:20]: Let's keep these paragraphs. It explains why you are using it with Palette in the quickstart and also provides a nice segue instead of jumping right in.

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (Siboney1) [reply content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:22]: This sentence refers to the steps Enabling a Nested Cluster as the document is about actions with Nested Clusters. I am describing steps to create a Nested Cluster. Steps to create a Host Cluster is described under Prerequisites.

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (Siboney1) [reply content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:37]: The "option" is mentioned in the previous sentence. the process is the same for both Nested and Host cluster deployment. Very clear.

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (Siboney1) [reply content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:22]: This step is for enabling a host cluster. The link gives them the instructions.

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (Siboney1) [reply content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:36]: Tyler updated this section to clarify.

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (Siboney1) [reply content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md]: Tyler updated this section to clarify.

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (Siboney1) [reply content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:20]: This is needed. If they arrive at this page from an internal or external search, it will give them a brief synposis of what this page is about and how it relates to their find. User flow is not always linear.

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (resmaa) [reply content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:22]: then frame it as :
Try the steps below to [enable](/clusters/nested-clusters/cluster-quickstart#enablinganestedclusteronahostcluster)  the host cluster and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a managed Nested Cluster from within Palette.

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/883#" title="Author self-approved">antongisli</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1107973081" title="Approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1109704412" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md
Comment (resmaa) [reply content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md:20]: they can always refer the overview, from the subpage

**Review by resmaa — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/883#" title="Author self-approved">antongisli</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1110060015" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1107973081" title="Approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1109704412" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by kreeuwijk — APPROVED** [review]
LGTM!

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/883#" title="Author self-approved">antongisli</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1112917430" title="Approved">kreeuwijk</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1110060015" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1107973081" title="Approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/883#pullrequestreview-1109704412" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
